.gitattributes

```diff
@@ -1,2 +1,4 @@
 * text eol=lf
 bin/just-1.14.0-x86_64-unknown-linux-musl binary
+bin/otel-cli binary
+bin/honeymarker binary
```

Comment:
You might have spotted this by the time you read this comment, but this line is no longer required.